### ISSUE TITLE
Zotero library only loaded (and stored) partially

### ISSUE THREAD

**Issue by jangenoe** [krassowski/jupyterlab-citation-manager#44]
I have  about 8000 references in my Zotero library, organized in collections. 
I see the progress indicator while loading the zotero items going to 100%.

However, only 875 items are found back in Jupyterlab.
This corresponds to a data block in my default jupyterlab workspace (default-xxxx.jupyterlab-workspace) about 512kB, where it is stored offline between sessions.

I can use the loaded 875 items in jupyterlab without any problem, but there seems no way to access my other +7000 Zotero references.

**Comment by krassowski** [comment]
Thanks for reporting. That's odd - there is a fallback value in case of the server misbehaving and not telling us how many items are there but it is 10000. 875 is an oddly specific number. If the progress bar reaches 100%, it would suggest that  `Total-Results` must be returning 875... or that there is a bug somewhere. Let's try to narrow it down:
- Are there any errors in the developer console in the browser?
- Are these citations all showing up correctly on zotero.org?

You could try to see how many items are returned by the zotero server by visiting:

```
https://api.zotero.org/users/<your user id>/items/?key=<your access key>
```

e.g. `https://api.zotero.org/users/475425/items/?key=P9NiFoyLeZu2bZNvvuQPDWsd` (that one does not work, but it works without a key as it is a public library for testing: https://api.zotero.org/users/475425/items/)

and checking the headers in the Network tab as on the GIF below:

![check-total-resuts](https://user-images.githubusercontent.com/5832902/148694212-f3cea9f6-6421-47e7-b04f-08ba98af7dc0.gif)

**Comment by jangenoe** [comment]
I start with the answer to the 2 questions on top:

* The developer console in the browser mentions mentions when the extension is loaded
```
Success fetching style elsevier-with-titles.csl from server extension   (i.e my default style)
Fetched 875 citable items from Zotero   (i.e. far too limited number of items)
```
* All the citations are showing up correctly in zotero.org

I did next also the same analysis as you did, and I get the ``totals results``=15653, see the screenshot below...
![image](https://user-images.githubusercontent.com/35977019/148699511-424ff075-6b7b-4128-9a89-836e32db28b5.png)
This is substantially above the 10000 number. But it does not at all correspond to the 875 that I get all the time.

**Comment by jangenoe** [comment]
I made a second [Zotero profile](https://www.zotero.org/support/kb/multiple_profiles), in which I stored a limited number of items (actually only comprising items I plan to cite from within jupyterlab).

**This works utterly smooth.**

This seems to confirm to issue is related to the size of the Zotero database (or a size limitation of its mirror offline copy inside jupyterlab).

**Comment by krassowski** [comment]
This should be fixed by #61 which is included in v0.3.2. Please reopen if the issue persists.
